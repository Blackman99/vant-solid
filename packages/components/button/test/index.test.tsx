import { render } from 'solid-testing-library'
import { describe, expect, test } from 'vitest'
import Button from '../src'

describe('Button', () => {
  test('render', () => {
    const { container, unmount } = render(() => <Button label="A Button" />)
    expect(container.innerHTML).toMatchSnapshot()
    unmount()
  })
})
