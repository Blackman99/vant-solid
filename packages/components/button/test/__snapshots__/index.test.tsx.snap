// Vitest Snapshot v1

exports[`Button > render 1`] = `"<button>A Button</button>"`;
